app/workers/evss/disability_compensation_form/submit_form526.rb

```diff
@@ -48,7 +48,7 @@ def response_handler(response)
       def retryable_error_handler(error)
         # update JobStatus, log and metrics in JobStatus#retryable_error_handler
         super(error)
-        raise EVSS::DisabilityCompensationForm::GatewayTimeout, error.message
```

Comment:
I don't understand why this choice was made originally.  I'd think we'd want to raise whatever the error really is.